mkdocs.yml

```diff
@@ -9,6 +9,8 @@ extra_javascript:
 markdown_extensions:
     - toc:
         anchorlink: true
+    - pymdownx.superfences
+    - admonition
```

Comment:
I just checked what these two extensions do and it [seems nice](https://squidfunk.github.io/mkdocs-material/extensions/admonition/)! (also for our whole spec, not just derivatives)

Yet I didn't find any part in the proposed changes where you make use of this currently. Did I miss it?


For reviewers who don't want to follow the link, it does some rendering like this:

![image](https://user-images.githubusercontent.com/9084751/82792210-54d49d80-9e6f-11ea-9e96-7f4ed7714baa.png)

Reply:
My guess is that #301 used it, but no longer did by the time it was merged. Would you prefer to remove it, or leave it in for future use?

Reply:
I would really like to make use of this in the future, but I am scared that it will break our PDF build. 

If we don't need it here though, we can drop it.